boards/remote-reva/include/periph_conf.h

```diff
@@ -99,6 +99,24 @@ static const periph_spi_conf_t spi_config[SPI_NUMOF] = {
 /** @} */
 
 /**
+ * @name ADC configuration
+ * @{
+ */
+#define ADC_NUMOF           (3)
+#define ADC1_PIN            5
+#define ADC2_PIN            4
+#define ADC3_PIN            2
+
+static const uint8_t periph_adc_map[ADC_NUMOF] = {
+  ADC1_PIN,
+  ADC2_PIN,
+  ADC3_PIN,
```

Comment:
Yes, this pin has a 5/3 voltage divider, meant to connect 5VDC sensors... I think I commented this somewhere in the platform, but it should be more explicit here, I will add the comment, thanks!